go.mod

```diff
@@ -229,8 +230,7 @@ replace (
 	// dgrijalva/jwt-go is deprecated and doesn't receive security updates.
 	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
-	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.0.0-20230605085654-2136ad029860
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.0.0-20230615084816-d14eadaa30e6
```

Comment:
in the long run, we should simply change the package path ;D

Reply:
not sure to which package?

Reply:
I mean import `github.com/crypto-org-chain/ethermint` directly, instead of adding replacements.